.github/workflows/autobuild.yml

```diff
@@ -1,110 +1,123 @@
 #### Automatically build and upload releases to GitHub ####
 
 on:
+  workflow_dispatch:
   push:
     tags:
-      - "r*" # run this action if a tag beginning with r is created
-  workflow_dispatch:
-
+    #  - "*"
```

Comment:
So, does it run on every push?

Reply:
for now yes. 
From my work I know this behaviour: always build, to get any build-error as early as possible. Publish the binaries only on request to a release.

discussion for #854, but I will disable it for now

Reply:
new intermediate behaviour: build on every tag, release on tag starting with "r"

Reply:
Building on every tag shouldn't be necessary (we already have CodeQl) this would waste resources (at least in my opinion)